java/BinExport/src/main/java/com/google/security/binexport/BinExport2Builder.java

```diff
@@ -311,8 +311,15 @@ private void buildFlowGraphs(Map<Long, Integer> basicBlockIndices)
               edge.setTargetBasicBlockIndex(targetId);
             }
             edges.add(edge.build());
+            // Possibly naive (and experimental) handling for switch tables
+          } else if (flow.isJump() && flow.isComputed()) {
+              edge.setSourceBasicBlockIndex(id);
```

Comment:
Please reformat these lines. The [Google Java-style](https://google.github.io/styleguide/javaguide.html) mandates two spaces for indentation.

To make this easier, you can install these formatter presets in your IDE:
* [Eclipse formatter preset](https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml)
* [IntelliJ formatter](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)